src/60_Sample_A4As/A4A_Banner_Ad.html

```diff
@@ -0,0 +1,43 @@
+<!--
+  #### Introduction
+
+  This is a simple banner [AMP Ad for AMP Pages](https://github.com/ampproject/amphtml/issues/3133) (A4A). In general an A4A should be a valid AMP document.
```

Comment:
With the new ABE design we show the first sample sentence in a card on the homepage. Suggestion: rephrase to "This samples demonstrates how to implement a simple banner [AMP Ad for AMP Pages](https://github.com/ampproject/amphtml/issues/3133) (A4A)."